shared/langs/en-us/starter-files.js

```diff
@@ -612,7 +612,7 @@ const MoreAlg = {
   "inequalities-compound": {
     "title": "Compound Inequalities Starter File",
     "pyret": {
-      "url": "https://code.pyret.org/editor#share=12dM89RqRWRoh51-saMBzLZyITbtrNQre"
+      "url": "https://code.pyret.org/editor#share=1wmfqAhQelP4L49nVuqosbfjcB0gpfS6T&v=0c33ff0"
```

Comment:
Reminder to remove the unused `&v=...` parameter from these links. It's totally unused but keeps changing when Ben and Joe roll out an update, which then causes unnecessary merge conflicts for us because the URL will be identical but the `&v=...` will be different.

Reply:
I had no idea.
@retabak , judging from the starterfiles.js file I just cleaned up looks like this will be news to you, too.